.github/workflows/ci.yml

```diff
@@ -317,6 +323,27 @@ jobs:
           make wasm
         if: steps.metadata.outputs.module == 'wasm'
 
+      ##
+      ## wasm-wasi-component
+      ##
+
+      - name: Setup rust
+        run: |
+          curl https://sh.rustup.rs | sh -s -- -y
+        if: steps.metadata.outputs.module == 'wasm-wasi-component'
+
+      - name: Configure wasm-wasi-component
+        run: |
+          ./configure wasm-wasi-component
+        if: steps.metadata.outputs.module == 'wasm-wasi-component'
+
+      - name: Make wasm-wasi-component
+        run: |
+          CLANG_PATH=/usr/bin/clang-15 \
+          BINDGEN_EXTRA_CLANG_ARGS="-I../../njs/src -I../../njs/build" \
+          make wasm-wasi-component
+        if: steps.metadata.outputs.module == 'wasm-wasi-component'
```

Comment:
Does it make sense just to include this in the `wasm` step?

Reply:
I did at first, but then thought it was sufficiently different...

Reply:
Cool! Looks to good to me 👍🏻